src/AgentPagesHandler.php

```diff
@@ -312,13 +312,11 @@ protected function agent()
     {
         global $wp_query;
 
-
-        //Strip the extra periods from the agentName
+        //Strip out extraneous periods and commas
         $agentName = preg_replace("/\./", "", $_REQUEST['agentId']);
+        $agentName = preg_replace("/,/", "", $agentName);
```

Comment:
@jonlipp You could combine the period and comma into one regular expression.

examples:
```javascript
/[\.,]/
```
```javascript
/(\.|,)/
```